web-ng/src/app/components/project-list/project-list.component.scss

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import '_variables';
+
+.project-card {
+  background-image: url(/assets/img/happy-land.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 160px;
+  height: 120px;
+  margin: 32px;
+  padding: 32px;
+}
+
+.recent-projects {
+  padding: 16px;
+}
+
+.card-content {
+  height: 100%;
+  width: 100%;
+}
+
+.project-add {
+  background-color: var(--white);
+  width: 160px;
+  height: 120px;
+  margin: 32px;
+  padding: 32px;
+}
+
+.logo {
+  width: 64px;
+  height: 64px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.title-project {
+  color: var(--white);
```

Comment:
Can we use material color here?

Reply:
https://github.com/google/ground-platform/pull/643#discussion_r674952590 yes @gino-m  asked to revert the changes